src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_FrameworkElement.cs

```diff
@@ -353,4 +377,16 @@ protected override Size MeasureOverride(Size availableSize)
 			return base.MeasureOverride(BaseAvailableSize ?? availableSize);
 		}
 	}
+
+	public partial class AspectRatioView : FrameworkElement
+	{
+		public double AspectRatio { get; set; } = 1.5;
+
+		protected override Size MeasureOverride(Size availableSize)
+		{
+			var height = double.IsPositiveInfinity(availableSize.Height) ? 0 : availableSize.Height;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'availableSize' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5173995)